constitution.tex

```diff
@@ -770,6 +770,16 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Waiving of Requirements}
+\begin{enumerate}
+  \item The following requirements may be waived as decided by a Two-Thirds Immediate Vote as described in \ref{Two-Thirds} and \ref{Immediate Vote}:
```

Comment:
You should note whether both requirements are always waived or if the vote must specify.
If the vote must specify, may both be waived at once?